src/components/Contacts/Contacts.module.css

```diff
@@ -0,0 +1,8 @@
+.wrap {
+	width: 600px;
+	background-color: white;
+	padding: 10px;
+	border: hidden;
+	border-radius: 10px;
+	box-shadow: 0 2px 4px rgba(0, 0, 0, .5);
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)